In [1]:
!date

Fri Jun 28 11:10:37 PDT 2019


In [2]:
!system_profiler SPSoftwareDataType

Software:

    System Software Overview:

      System Version: macOS 10.14.5 (18F132)
      Kernel Version: Darwin 18.6.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Computer Name: Megan’s MacBook Pro (2)
      User Name: Megan Duffy (meganduffy)
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled
      Time since boot: 5 days 15:14



### Notebook to document Unipept least common ancestor analysis of UWPR October 2018 ETNP 2017 (samples 2-9 and 4-19) de novo only peptides (dno)
### Input: results of PEAKS 8.5 searches into Git ETNP 2017 data directory. Raw data processed to give dno peptides without modifications (for east of analysis). See uwpr-apr2017-dno [notebook](https://github.com/MeganEDuffy/2017-etnp/blob/master/notebooks/uwpr-apr2017-dno.ipynb).

In [4]:
# Example of data file types in data directory

!ls /Users/meganduffy/Documents/git-repos/2017-etnp/data/uwpr-apr2017/UWPR_Apr2017_231-100m-ETNP_PEAKS_97/

231-peaks97-DB-search-psm.csv    231-peaks97-dno.csv
231-peaks97-db-mods.txt          231-peaks97-dnopeps.csv
231-peaks97-db-nomods.txt        231-peaks97-dnopeps.txt
231-peaks97-dbpeps.csv           231-peaks97-peptide.csv
231-peaks97-dbpeps.txt           231-peaks97-protein-peptides.csv
231-peaks97-dno-mods.txt         231-peaks97-proteins.csv
231-peaks97-dno-nomods.txt       231-peaks97-proteins.fasta


### First, analysing sample from this batch that's relevant to Pro Novo paper: 231 (100 m suspended).
### Ran 231-peaks97-dno-nomods.txt through Unipept metaproteomics analysis web server. Equated L and I and filtered duplicate peptides.
### 1392/3218 peptides found matching to Unipept DB
### Uniqueness: 3218/3437 peptides unique (duplicates filter)
### Exported results to .csv in git analyses directory, and saved sunburst figure as .png

In [5]:
!ls /Users/meganduffy/Documents/git-repos/2017-etnp/analyses/uwpr-apr2017/unipept-lca/

231-peaks97-dno-nomods-lca.csv 231-peaks97-dno-nomods-lca.png


### Note: seven peptides matched to Cyanobacteria

### Sample 233 (265 m suspended)

In [6]:
!ls /Users/meganduffy/Documents/git-repos/2017-etnp/data/uwpr-apr2017/UWPR_Apr2017_233-265m-ETNP_PEAKS_105/

233-peaks105-DB-search-psm.csv    233-peaks105-dno.csv
233-peaks105-db-mods.txt          233-peaks105-dnopeps.csv
233-peaks105-db-nomods.txt        233-peaks105-dnopeps.txt
233-peaks105-dbpeps.csv           233-peaks105-peptide.csv
233-peaks105-dbpeps.txt           233-peaks105-protein-peptides.csv
233-peaks105-dno-mods.txt         233-peaks105-proteins.csv
233-peaks105-dno-nomods.txt       233-peaks105-proteins.fasta


### Ran 233-peaks105-dno-nomods.txt through Unipept metaproteomics analysis web server. Equated L and I and filtered duplicate peptides.
### 1129/2658 peptides found matching to Unipept DB
### Uniqueness: 2658/2854 peptides unique (duplicates filter)
### Exported results to .csv in git analyses directory, and saved sunburst figure as .png

### Note: two peptides matched to Cyanobacteria